lib/assessment.rb

```diff
@@ -20,20 +30,35 @@
 # Name and location should be write-only, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_reader :age
+end
+
+def initialize(name, age, _location)
+  @name = name
+  @age = age
+  @location = age
+end
+
 ##
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
+Person.new = ('Dave', 32, 'Somerville')
 
-Answer.dave = nil # replace nil with your code
-Answer.dave.location = nil # replace nil with your code
+
+Answer.dave =  "Dave"# replace nil with your code
```

Comment:
"Dave" is not a reference. What code should be to the right of the assignment operator?